.zenodo.json

```diff
@@ -3,15 +3,16 @@
     "description": "SFINCS plugin for HydroMT.",
     "upload_type": "software",
     "creators": [
-        {
-            "affiliation":"Deltares",
-            "name": "de Goede, Roel"
-        },
         {
             "affiliation": "Deltares",
             "name": "Eilander, Dirk",
             "orcid": "0000-0002-0951-8418"
         },
+        {
+            "affiliation":"Deltares",
```

Comment:
happy to have you as first author on this software! Should we also add Maarten and Kees?

Reply:
Could you / did you also carefully compare this file against the hydromt core file to make sure it works?

Reply:
Wouldn't that be confusing, since it has been Eilander et al for a while?

File seems similar now to the core (though that already was the case), except the licensing.

Reply:
I think last time it failed on the license. It's hard to find good documentation about accepted license strings. I've changed based on another example I found which seemed to work.